packages/grid/docker-compose.yml

```diff
@@ -141,6 +128,7 @@ services:
     env_file:
       - .env
     environment:
+      - RELEASE=${RELEASE?production}
```

Comment:
```suggestion
      - RELEASE=${RELEASE:-production}
```

Reply:
I will check this out and try to update all usages like this because Madhava said he meant to set `production` as a default value in case `RELEASE` is unset or null.

Reply:
Actually I can't make changes to the branch of this PR because I don't have push permissions for OpenMined/PySyft which is required as explained in https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/committing-changes-to-a-pull-request-branch-created-from-a-fork

Reply:
Opened another PR with my suggestion on top of this one. It's referenced below https://github.com/OpenMined/PySyft/pull/6214